### PR TITLE
feat: Expose the backend as an SDK integration

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

File: tests/assertions.py [hunk 1]
```diff
@@ -33,16 +33,15 @@ def assert_meta(envelope, release="test-example-release"):
...
-        },
+    }
+    expected_sdk = {
+        "name": "sentry.native",
+        "version": "0.4.2",
+        "packages": [{"name": "github:getsentry/sentry-native", "version": "0.4.2"},],
     }
 
     assert matches(event, expected)
+    assert matches(event["sdk"], expected_sdk)
```
Comment (flub) [inline tests/assertions.py:44]: I don't really get the test.  You're not actually seeing the backend thing you just added but just re-wrote this a bit?

Comment (Swatinem) [reply tests/assertions.py:44]: I basically want to exclude that from the match, as the assertion fails if we have additional attributes. And I don’t really want to pass in that flag explicitly for each testcase.

Comment (flub) [reply tests/assertions.py:44]: Is it possible to add a test that checks it's present but `in ["crashpad", "breakpad", "inproc"]`?

Comment (Swatinem) [reply tests/assertions.py:44]: its not present for "none" ;-)

Comment (jan-auer) [reply tests/assertions.py:44]: Can you then at least test it in a separate assertion?

**Review by jan-auer — APPROVED** [review]

**Pull request merged**